### PR TITLE
Lesson 5 task 1

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Text class created with methods to check if text is palindrome

### CODE REVIEW

File: src/main/java/lesson5/task1/Text.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        if (inputText.isEmpty()) {
+            return "Text is Empty!";
+        } else if (isPalindrome(inputText)) {
+            return "Text is a palindrome and the length of the text is " + inputText.length();
+        } else {
+            return "Text is not a palindrome!";
+        }
+    }
+
+    public boolean isPalindrome(String newText) {
```
Comment (strzeleckik) [inline src/main/java/lesson5/task1/Text.java:21]: It would be good to add some unit tests here

**Review by strzeleckik — CHANGES_REQUESTED** [review]
I gave you feedback

File: src/main/java/lesson5/task1/Text.java (hunk 1 above)
Comment (jean-farinea) [reply src/main/java/lesson5/task1/Text.java:21]: Unit test class created. Please review.

**Review by strzeleckik — APPROVED** [review]
If you wrote test in this way, you should create two test methods, one for positive scenario and second for negative scenario, just a hint for you for future tasks

**Comment by jean-farinea** [comment]
Test class adjusted for negative and positive cases. 
*Wrong naming at commit!

**Pull request merged**